api/v6/report_server.thrift

```diff
@@ -127,10 +127,20 @@ struct RunHistoryData {
   2: string    runName,     // Name of the run.
   3: string    versionTag,  // Version tag of the report.
   4: string    user,        // User name who analysed the run.
-  5: string    time         // Date time when the run was analysed.
+  5: string    time,        // Date time when the run was analysed.
+  6: i64       id,          // Id of the run history tag.
 }
 typedef list<RunHistoryData> RunHistoryDataList
 
+/**
+ * Members of this struct are interpreted in "AND" relation with each other.
+ * Between the list elements there is "OR" relation.
+ * If exactMatch field is True it will use exact match for run names.
+ */
+struct TagFilter {
```

Comment:
Later this way we can add an option easily to filter run history data by `author` or `date`. I have also renamed it to `RunHistoryFilter` because it is much more expressive.